www-client/links/patches/links-2.28.patchset

```diff
@@ -27,5 +27,35 @@ index 8031d8f..f21b8f2 100644
  	switch (msg->what) {
  	case B_MOUSE_WHEEL_CHANGED:
 -- 
-2.28.0
+2.37.3
+
+
+From 7ecbf6ae4d023e655449c0596e644a29da12e6e7 Mon Sep 17 00:00:00 2001
+From: Ken Mays <kmays2000@gmail.com>
+Date: Mon, 12 Dec 2022 04:12:05 +1000
+Subject: Disabled ClipBoard Text for Haiku
```

Comment:
Why?

Reply:
Need to provide updated patch for upstream fix.

Reply:
That doesn't answer my question.

Reply:
My question is - is the current clipboard set/get code used within haiku.cpp 'currently' correct? Seems to break the compiler build on nightly - so I currently disabled it until verification.

Reply:
https://github.com/haikuports/haikuports/issues/7555

Reply:
What is the error?

Reply:
Updated in ticket.